runtime/vara/src/lib.rs

```diff
@@ -1392,3 +1397,60 @@ impl_runtime_apis_plus_common! {
         }
     }
 }
+
+#[cfg(any(feature = "std", test))]
+impl<B, C> Clone for RuntimeApiImpl<B, C>
+where
+    B: BlockT,
+    C: CallApiAt<B>,
+    C::StateBackend: StateBackend<HashFor<B>>,
+    <C::StateBackend as StateBackend<HashFor<B>>>::Transaction: Clone,
+{
+    fn clone(&self) -> Self {
+        Self {
+            call: <&C>::clone(&self.call),
+            commit_on_success: self.commit_on_success.clone(),
+            changes: self.changes.clone(),
+            storage_transaction_cache: self.storage_transaction_cache.clone(),
+            recorder: self.recorder.clone(),
+        }
+    }
+}
+
+#[cfg(any(feature = "std", test))]
+impl<B, C> common::RuntimeApiExt<C> for RuntimeApiImpl<B, C>
```

Comment:
```suggestion
/// This trait provides methods to deconstruct and restore the `RuntimeApiImpl` struct.
///
/// substrate/primitives/api/proc-macro/src/impl_runtime_apis.rs:219
impl<B, C> common::RuntimeApiExt<C> for RuntimeApiImpl<B, C>
```

Or insert code ref to `common::RuntimeApiExt`